.code-samples.meilisearch.yaml

```
@@ -393,6 +393,16 @@ settings_guide_displayed_1: |-
             "release_date"
         ]
     }'
+settings_guide_sortable_1: |-
+  curl \
+    -X POST 'http://localhost:7700/indexes/movies/settings' \
```

Since the example attributes (`price` and `author`) refer to our bookshop examples, perhaps it makes more sense to use a `books` index?

```suggestion
    -X POST 'http://localhost:7700/indexes/books/settings' \
```

---

Ah yes, thanks for catching this.